.github/workflows/test.yml

```
@@ -215,6 +215,135 @@ jobs:
+       # ports: 
+        #  - 3306:3306
+        options: --health-cmd="mysqladmin ping"
+                 --health-interval=5s
+                 --health-timeout=5s
+                 --health-retries=3
+    steps:
+      - name: get mariadb container id
+        run: echo "::set-output name=id::$(docker container ls | grep mariadb | awk '{ print $1 }')"
+        id: mariadb_container
+      - name: show networks
+        run: echo "$(docker network ls)"
+      - name: get automatic created network
+        run: echo "::set-output name=id::$(docker network ls | grep github_network | awk '{ print $1 }')"
+        id: network
+      - name: Start Login-Server
+        run: docker run --network ${{ steps.network.outputs.id }} --name=login-server -d gradido/login_server:default
+      - name: get login-server container id
+        run: echo "::set-output name=id::$(docker container ls | grep login_server | awk '{ print $1 }')"
+        id: login_server_container
+      ##########################################################################
+      # CHECKOUT CODE ##########################################################
+      ##########################################################################
+      - name: Checkout code
+        uses: actions/checkout@v2
+      ##########################################################################
+      # DOWNLOAD DOCKER IMAGE ##################################################
+      ##########################################################################
+      - name: Download Docker Image (Community-Server)
+        uses: actions/download-artifact@v2
+        with:
+          name: docker-community-server-test
+          path: /tmp
+      - name: Load Docker Image
+        run: docker load < /tmp/community_server.tar
+        
+      # for debugging login-server
+      - name: check login-server
+        run: docker logs ${{ steps.login_server_container.outputs.id }}
+      - name: check mariadb
+        run: docker logs ${{ steps.mariadb_container.outputs.id }}
+      ##########################################################################
+      # UNIT TESTS BACKEND COMMUNITY-SERVER  #######################################
+      ##########################################################################
+      - name: community server | Unit tests
+        run: |
+          docker run --network ${{ steps.network.outputs.id }} -v ~/coverage:/var/www/cakephp/webroot/coverage gradido/community_server:test
+          cp -r ~/coverage ./coverage
+      #########################################################################
+      # COVERAGE CHECK BACKEND COMMUNITY-SERVER ####################################
+      ##########################################################################
+      - name: backend community simplecov | Coverage check
+        uses: webcraftmedia/coverage-check-action@master
+        with:
+          report_name: Coverage Backend
+          type: simplecov
```

```suggestion
          type: lcov
```

What is the error when you do this?